db/mysql.js

```diff
@@ -96,7 +96,7 @@ module.exports = function (log, error) {
         free: 0
       }
     )
-    log.stat(stats)
+    log.info('stats', stats)
   }
```

Comment:
mozlog doesn't have a `.stat()`, so I changed it to an `.info()`. Anyone see any problem doing this?